src/libcore/lib.rs

```diff
@@ -23,12 +23,6 @@
 //! nor does it provide concurrency or I/O. These things require
 //! platform integration, and this library is platform-agnostic.
 //!
-//! *It is not recommended to use the core library*. The stable
```

Comment:
Well that's kinda true but also why the facade is controversial. I do agree that explaining more of the divide here would be good, but in this patch, I'm more focused on removing inaccuracy than I am making these docs great.